README.md

```diff
@@ -1,4 +1,4 @@
-[![Code Climate](https://codeclimate.com/badge.png)](https://codeclimate.com/github/thibaudgg/rb-fsevent)
+[![Code Climate](https://codeclimate.com/github/thibaudgg/rb-fsevent.svg)](https://codeclimate.com/github/thibaudgg/rb-fsevent)
```

Comment:
Shouldn't we use `guard/rb-fsevent` here instead as it's the new GitHub address?

```suggestion
[![Code Climate](https://codeclimate.com/guard/rb-fsevent/rb-fsevent.svg)](https://codeclimate.com/guard/rb-fsevent/rb-fsevent)
```

Reply:
Hey, @thibaudgg! I've overlooked it's now located at the new GitHub address, but my change is the right one.
First of all, the proper links in this case would be https://codeclimate.com/github/guard/rb-fsevent and https://codeclimate.com/github/guard/rb-fsevent.svg, but they both return 404s right now, as the repo was originally set up on CodeClimate as `thibaudgg/rb-fsevent` and it seems like the service won't transfer the project to another account automatically.

So unless you want these links to refer to the new location then my change is sufficient. Otherwise it may be that this repo needs to be set up on CodeClimate by the authorized user/owner of the `guard` organization.

Reply:
Good point, I just reset the CodeClimate remote, with that code it should work now:

```
[![Code Climate Maintainability](https://api.codeclimate.com/v1/badges/75c32e6c747552409332/maintainability)](https://codeclimate.com/github/guard/rb-fsevent/maintainability)
```

Reply:
Awesome! And now that you've reset the CodeClimate remote, we can update the links pointing to the `guard/rb-fsevent` repo! 🥳 I just added a fixup, thank you!

Reply:
Great, thanks to you!